modules/compliance-supported-profiles.adoc

```diff
@@ -132,4 +132,41 @@ The Compliance Operator provides the following compliance profiles:
 |===
 [.small]
 1. To locate the CIS {product-title} v4 Benchmark, go to  link:https://www.cisecurity.org/benchmark/kubernetes[CIS Benchmarks] and click *Download Latest CIS Benchmark*, where you can then register to download the benchmark.
-2. Node profiles must be used with the relevant Platform profile. For more information, see xref:../../security/compliance_operator/compliance-operator-understanding.adoc#compliance_profile_typesunderstanding-compliance[Compliance Operator profile types].
\ No newline at end of file
+2. Node profiles must be used with the relevant Platform profile. For more information, see xref:../../security/compliance_operator/compliance-operator-understanding.adoc#compliance_profile_typesunderstanding-compliance[Compliance Operator profile types].
+
+[id="compliance-extended-profiles_{context}"]
+== About extended compliance profiles
+
+Some compliance profiles are extensions of other profiles, building on an existing rule set. For example, NIST 800-53 High-Impact Baseline is an extension of the CIS Red Hat OpenShift Container Platform 4 profile.
```

Comment:
I think that's great @rhmdnd . Would this be a great time to link externally, like: https://listings.pcisecuritystandards.org/pdfs/Mapping-PCI-DSS-to-NIST-Framework-At-a-Glance.pdf ?

Reply:
@rhmdnd what do you think about: 
```Some compliance profiles are extensions of other profiles, building on security best practices provided by the Center for Internet Security (CIS). Combining CIS best practices with National Institute of Standards and Technology (NIST) security frameworks establishes a path to a secure and compliant environment.```

Reply:
The only tweak I would make would be to say some baselines have controls that require following industry best practices, which results in some profiles extending others. For example, this is why the PCI-DSS profile and NIST baselines extend the CIS OpenShift profile. Otherwise, it sounds like PCI-DSS points to NIST, which points to CIS? Is that correct?

Reply:
That's perfect. I knew I was missing a part of it, just didn't know what :smile: I removed PCI-DSS from the example to avoid further confusion. But it is interesting to me that it extends both NIST moderate and CIS. 

How about this:

Some compliance profiles have controls that require following industry best practices, resulting in some profiles extending others.  Combining the Center for Internet Security (CIS) best practices with National Institute of Standards and Technology (NIST) security frameworks establishes a path to a secure and compliant environment.

For example, the NIST High- and Moderate-Impact profiles extend CIS best practices to achieve compliance.